docs/release-notes/2.11.0.asciidoc

```diff
@@ -0,0 +1,68 @@
+:issue: https://github.com/elastic/cloud-on-k8s/issues/
+:pull: https://github.com/elastic/cloud-on-k8s/pull/
+
+[[release-notes-2.11.0]]
+== {n} version 2.11.0
+
+
+
+[[feature-2.11.0]]
+[float]
+=== New features
+
+* Introduce Kibana config field in stack config policy {pull}7324[#7324]
+* Add StatefulSet as a deployment option for Elastic Agent {pull}7357[#7357]
```

Comment:
There should also be a PR to add config field and secretMounts field to the Elasticsearch field in scp
https://github.com/elastic/cloud-on-k8s/pull/7233

Reply:
I initially removed this, as I saw this as a part of the bigger feature for the stack config policy changes, but I think that's for the highlights, not release notes. I've added it back.